server/atlassian_connect.go

```diff
@@ -41,13 +42,15 @@ func renderAtlassianConnectJSON(w http.ResponseWriter, r *http.Request, _ *Plugi
 	}
 	tmpl, err := template.ParseFiles(tmplPath)
 	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+		config.Mattermost.LogWarn("Error parsing the template", "error", err.Error())
+		http.Error(w, "error rendering the template", http.StatusInternalServerError)
 		return
 	}
 
 	w.Header().Set("Content-Type", "application/json")
 	if err = tmpl.Execute(w, values); err != nil {
-		http.Error(w, "failed to write response: "+err.Error(), http.StatusInternalServerError)
+		config.Mattermost.LogWarn("Error writng the template as response", "error", err.Error())
```

Comment:
Any time we return a 500 http error on the response, the logger level should be `error`.